test/unit/test_compose.js

```diff
@@ -729,29 +735,32 @@ TD.commonCase('reply/forward html message', function(T, RT) {
       html: replyHtmlHtml.replace('$MESSAGEID$', header.guid)
     };
 
-    eCheck.expect_namedValue('reply text', expectedReplyBody.text);
-    eCheck.expect_namedValue('reply html', expectedReplyBody.html);
+    eCheck.expect('reply text',  expectedReplyBody.text);
+    eCheck.expect('reply html',  expectedReplyBody.html);
 
-    eCheck.expect_event('sent');
+    eCheck.expect('sent');
 
     header.replyToMessage('sender', function(composer) {
-      eCheck.namedValue('reply text', composer.body.text);
-      eCheck.namedValue('reply html', composer.body.html);
+      eCheck.log('reply text', composer.body.text);
+      eCheck.log('reply html', composer.body.html);
 
       composer.finishCompositionSendMessage();
 
       testUniverse.MailAPI.onbackgroundsendstatus = function(data) {
         if (data.state === 'success') {
-          eCheck.event('sent');
+          eCheck.log('sent');
         }
       };
     });
   });
+
+  var asyncLazy = T.lazyLogger();
+
   testAccount.do_waitForMessage(inboxView, 'Re: ' + uniqueSubject, {
     expect: function() {
       // We are going to want to add some expectations in the withMessage case
       // so avoid early resolution of the account's logs.
-      testAccount.eOpAccount.asyncEventsAreComingDoNotResolve();
+      asyncLazy.expect('asyncEventsReceived');
```

Comment:
It's awesome that the unified eventListener approach in legacy_gelamtest.js eliminates the need for the asyncEventsAreComingDoNotResolve stuff.  (The loggest stuff was a bit promise-happy!)